decidim-proposals/db/migrate/20171212102250_enable_pg_extensions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class EnablePgExtensions < ActiveRecord::Migration[5.1]
+  def change
+    enable_extension "pg_trgm"
```

Comment:
@decidim/lot-core @decidim/product To activate the extension the user of the database needs SUPERUSER permissions, perhaps this migration is unnecessary if in the requirements of `decidim` and/or` decidim-proposals` is specified that `pg` requires the extension `pg_trgm` to find similar proposals.

Reply:
It should be safe to add this extension. In case the user doesn't have permissions, you can always modify the migration in your application and enable it manually. Specifying it will remove the need to explicitly document it as a prerequisite (as it will either just work or blow up).

Reply:
@josepjaume ok I leave it as it is now.

Reply:
I usually do it like this: https://github.com/alabs/nolotiro.org/blob/52fb039c0f159d970f225d02b78533c790d9f453/db/migrate/20161117184157_enable_indexes_on_like_queries.rb